### PR TITLE
For Technical Review - GSQ Dataset Themes

### PR DESCRIPTION
**Pull request by KellyVance**
For technical review of ttl structure, composition, and format

### CODE REVIEW

File: vocabularies/gsq-dataset-themes.ttl [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+:conceptScheme a skos:ConceptScheme ;
+    dct:created "2019-10-02T15:00:47+10:00"^^xsd:dateTime ;
+    dct:creator "Geological Survey of Queensland" ;
+    dct:modified "2019-10-02T15:00:47+10:00"^^xsd:dateTime ;
+    dct:source "Developed by the Geological Survey of Queensland." ;
+    skos:definition "A list of broad thematic descriptions of datasets created, received, and held by the Geological Survey of Queensland."@en ;
+    skos:prefLabel "GSQ Dataset Themes"@en .
+
+:administrative-data a skos:Concept ;
+    skos:definition "Ancillary information regarding the administration of Queensland Resources such as adminstrivite boundaries, minng permits, legislation, and information about organisations such as DNRME and GSQ."@en ;
```
Comment (DavidCrosswellGSQ) [inline vocabularies/gsq-dataset-themes.ttl]: Two typos in this definition: administrative and mining.

File: vocabularies/gsq-dataset-themes.ttl [hunk 2]
```diff
@@ -0,0 +1,100 @@
+@prefix : <http://linked.data.gov.au/def/gsq-dataset-themes/> .
```
Comment (DavidCrosswellGSQ) [inline vocabularies/gsq-dataset-themes.ttl]: Still not a fan of using just ```:``` as the prefix. A short prefix makes it easier to read for me.

File: vocabularies/gsq-dataset-themes.ttl [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+    skos:topConceptOf :conceptScheme .
+
+:operational-data a skos:Concept ;
+    skos:definition "Data describing the operations of mining, drilling, processing, and sales operations e.g. borehole drilling, pipelines, processing facilities."@en ;
+    skos:inScheme :conceptScheme ;
+    skos:prefLabel "Operational data"@en ;
+    skos:topConceptOf :conceptScheme .
+
+:physical-samples a skos:Concept ;
+    skos:definition "Data and metadata pertaining to physical samples taken during geoscience data acquisition e.g. dril core, rock chips, rock samples from outcrop, soil, sediment, water, drilling fluids, oil, gas etc."@en ;
```
Comment (LukeHauck) [inline vocabularies/gsq-dataset-themes.ttl]: drill core typo in definition - "dril core"

File: vocabularies/gsq-dataset-themes.ttl [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+<http://linked.data.gov.au/def/gsq-dataset-themes> a owl:Ontology .
+
+:conceptScheme a skos:ConceptScheme ;
+    dct:created "2019-10-02T15:00:47+10:00"^^xsd:dateTime ;
+    dct:creator "Geological Survey of Queensland" ;
+    dct:modified "2019-10-02T15:00:47+10:00"^^xsd:dateTime ;
+    dct:source "Developed by the Geological Survey of Queensland." ;
+    skos:definition "A list of broad thematic descriptions of datasets created, received, and held by the Geological Survey of Queensland."@en ;
+    skos:prefLabel "GSQ Dataset Themes"@en .
```
Comment (DavidCrosswellGSQ) [inline vocabularies/gsq-dataset-themes.ttl]: Should this be "GSQ dataset themes"?

File: vocabularies/gsq-dataset-themes.ttl [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+    skos:topConceptOf :conceptScheme .
+
+:reports a skos:Concept ;
+    skos:definition "Statutory reports lodged for compliance with tenure grant and retention. These may be tenure based such as a resource authority annual report or activity based such as a well completion report."@en ;
+    skos:inScheme :conceptScheme ;
+    skos:prefLabel "Reports"@en ;
+    skos:topConceptOf :conceptScheme .
+
+:resource-statistics a skos:Concept ;
+    skos:definition "Statistical data and reports submitted to the department or collected/generated by the department, particulary production and sales quantities."@en ;
```
Comment (LukeHauck) [inline vocabularies/gsq-dataset-themes.ttl]: particularly typo in definition - "particulary"

File: vocabularies/gsq-dataset-themes.ttl [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+    skos:topConceptOf :conceptScheme .
+
+:spatial-data a skos:Concept ;
+    skos:definition "Data and metadata related to spatial objects and representations such as maps, aerial and satellite imagery, spatial data files and databases."@en ;
+    skos:inScheme :conceptScheme ;
+    skos:prefLabel "Spatial data"@en ;
+    skos:topConceptOf :conceptScheme .
+
+:activities-and-results a skos:Collection ;
+    skos:definition "Primary dataset that describe the geological features of interest and the activities that generate those datasets"@en ;
```
Comment (LukeHauck) [inline vocabularies/gsq-dataset-themes.ttl]: For consistency, add Full Stop at end of definition text.

File: vocabularies/gsq-dataset-themes.ttl [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+    skos:topConceptOf :conceptScheme .
+
+:publications a skos:Concept ;
+    skos:definition "Publications that describe the geology, resource potential, or other areas of departmental interest such as books, journals, periodicals, presentations, posters, academic publications, or multimedia products."@en ;
+    skos:inScheme :conceptScheme ;
+    skos:prefLabel "Publications"@en ;
+    skos:topConceptOf :conceptScheme .
+
+:reports a skos:Concept ;
+    skos:definition "Statutory reports lodged for compliance with tenure grant and retention. These may be tenure based such as a resource authority annual report or activity based such as a well completion report."@en ;
```
Comment (DavidCrosswellGSQ) [inline vocabularies/gsq-dataset-themes.ttl]: Replace "tenure" with "permit"?

Not just statutory reports. Could be reports generated by GSQ or others.

File: vocabularies/gsq-dataset-themes.ttl [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+    skos:prefLabel "Activities and Results"@en .
+
+:ancillary-information a skos:Collection ;
+    skos:definition "Data descriptive of the organisations involved in data collection and the administrative constraints and associations under which data is collected or related."@en ;
+    skos:member :administrative-data,
+        :spatial-data ;
+    skos:prefLabel "Ancillary Information"@en .
+
+:products-and-documents a skos:Collection ;
+    skos:definition "The final products and official reports that synthesis the collected data into documentation that describe activities and/or provide interpretive and contextual information."@en ;
```
Comment (LukeHauck) [inline vocabularies/gsq-dataset-themes.ttl]: In the definition, should the word be "synthesise" not "synthesis"

Comment (DavidCrosswellGSQ) [inline vocabularies/gsq-dataset-themes.ttl]: Typo interpretative

File: vocabularies/gsq-dataset-themes.ttl [hunk 9]
```diff
@@ -0,0 +1,100 @@
...
+    skos:prefLabel "Spatial data"@en ;
+    skos:topConceptOf :conceptScheme .
+
+:activities-and-results a skos:Collection ;
+    skos:definition "Primary dataset that describe the geological features of interest and the activities that generate those datasets"@en ;
+    skos:member :geoscience-data,
+        :geoscience-project-data,
+        :operational-data,
+        :physical-samples ;
+    skos:prefLabel "Activities and Results"@en .
```
Comment (DavidCrosswellGSQ) [inline vocabularies/gsq-dataset-themes.ttl]: Activities and results

File: vocabularies/gsq-dataset-themes.ttl [hunk 10]
```diff
@@ -0,0 +1,100 @@
...
+
+:ancillary-information a skos:Collection ;
+    skos:definition "Data descriptive of the organisations involved in data collection and the administrative constraints and associations under which data is collected or related."@en ;
+    skos:member :administrative-data,
+        :spatial-data ;
+    skos:prefLabel "Ancillary Information"@en .
```
Comment (DavidCrosswellGSQ) [inline vocabularies/gsq-dataset-themes.ttl]: Ancillary information

File: vocabularies/gsq-dataset-themes.ttl [hunk 11]
```diff
@@ -0,0 +1,100 @@
...
+
+:products-and-documents a skos:Collection ;
+    skos:definition "The final products and official reports that synthesis the collected data into documentation that describe activities and/or provide interpretive and contextual information."@en ;
+    skos:member :publications,
+        :reports ;
+    skos:prefLabel "Products and Documents"@en .
```
Comment (DavidCrosswellGSQ) [inline vocabularies/gsq-dataset-themes.ttl]: Products and documents

File: vocabularies/gsq-dataset-themes.ttl (hunk 2 above)
Comment (KellyVance) [reply vocabularies/gsq-dataset-themes.ttl]: can add in a fix to the template. But as it is a style issue. I'm comfortable leaving as is for this

File: vocabularies/gsq-dataset-themes.ttl (hunk 1 above)
Comment (KellyVance) [reply vocabularies/gsq-dataset-themes.ttl]: fixed

File: vocabularies/gsq-dataset-themes.ttl (hunk 3 above)
Comment (KellyVance) [reply vocabularies/gsq-dataset-themes.ttl]: fixed

File: vocabularies/gsq-dataset-themes.ttl (hunk 8 above)
Comment (KellyVance) [reply vocabularies/gsq-dataset-themes.ttl]: I'd prefer Australia be more British than American but...

Interpretative vs. interpretive
Interpretative is slightly more common than interpretive in 21st-century British publications. Everywhere else, including in the U.S., Canada, and _Australia_, the shorter form is preferred.

https://grammarist.com/usage/interpretative-interpretive/

File: vocabularies/gsq-dataset-themes.ttl (hunk 9 above)
Comment (KellyVance) [reply vocabularies/gsq-dataset-themes.ttl]: fixed

File: vocabularies/gsq-dataset-themes.ttl (hunk 10 above)
Comment (KellyVance) [reply vocabularies/gsq-dataset-themes.ttl]: fixed

File: vocabularies/gsq-dataset-themes.ttl (hunk 11 above)
Comment (KellyVance) [reply vocabularies/gsq-dataset-themes.ttl]: fixed

File: vocabularies/gsq-dataset-themes.ttl (hunk 5 above)
Comment (KellyVance) [reply vocabularies/gsq-dataset-themes.ttl]: fixed

File: vocabularies/gsq-dataset-themes.ttl (hunk 6 above)
Comment (KellyVance) [reply vocabularies/gsq-dataset-themes.ttl]: fixed

File: vocabularies/gsq-dataset-themes.ttl (hunk 8 above)
Comment (KellyVance) [reply vocabularies/gsq-dataset-themes.ttl]: fixed

File: vocabularies/gsq-dataset-themes.ttl (hunk 4 above)
Comment (KellyVance) [reply vocabularies/gsq-dataset-themes.ttl]: Should it? We do mention Queensland, DNRME, GSQ etc.  but we could replace these with generic 'regulatory body' type terms.

File: vocabularies/gsq-dataset-themes.ttl (hunk 7 above)
Comment (KellyVance) [reply vocabularies/gsq-dataset-themes.ttl]: defintion changed to reflect broader scope

File: vocabularies/gsq-dataset-themes.ttl (hunk 4 above)
Comment (DavidCrosswellGSQ) [reply vocabularies/gsq-dataset-themes.ttl]: It was the capitalisation - not changing it to any other term. I should have been clearer :)

File: vocabularies/gsq-dataset-themes.ttl (hunk 8 above)
Comment (DavidCrosswellGSQ) [reply vocabularies/gsq-dataset-themes.ttl]: I don't mind either way. https://www.macquariedictionary.com.au/features/word/search/?search_word_type=Dictionary&word=interpretive

File: vocabularies/gsq-dataset-themes.ttl (hunk 4 above)
Comment (KellyVance) [reply vocabularies/gsq-dataset-themes.ttl]: doh! fixed. BUT GSQ kept capitals as it is an abbreviated set of initials

**Pull request merged**